### ISSUE TITLE
detecting a large eukaryotic genome

### ISSUE THREAD

**Issue by kternus** [sourmash-bio/sourmash#492]
I was wondering how sourmash could be used to identify a large eukaryotic genome within a metagenome? 

For example, detecting sequences from a specific species of fish within an aquatic metagenome.

**Comment by ctb** [comment]
right, here we want converse functionality from `gather`, e.g. a command like:

```
sourmash search --containment fish.sig metagenome1.sig metagenome2.sig
```

would work, but maybe with different output columns.

**Comment by ctb** [comment]
`sourmash prefetch` will work for detecting this! https://github.com/dib-lab/sourmash/pull/1370#issuecomment-833944673

**Comment by ctb** [comment]
we now have multiple commands and tutorials for this!

see https://github.com/sourmash-bio/sourmash/issues/3237 for a (now out of date 😭 ) tutorial and also `manysearch` from https://github.com/sourmash-bio/sourmash_plugin_branchwater.
